packages/babel-generator/test/fixtures/sourcemaps/comment-before-parentheses-return-arg-createParenthesizedExpressions/output.js

```diff
@@ -0,0 +1,17 @@
+function assertElement(assertFn, shouldBeElement, opt_message) {
+  return (/** @type {!Ele	ment} */(
+    assertType_(
+    assertFn,
```

Comment:
Probably unrelated, but the indentation here is broken.

Reply:
Yes, I plan to take a look at this in a follow-up PR. :)